src/main/java/org/traccar/protocol/GatorProtocolDecoder.java

```diff
@@ -90,7 +101,51 @@ protected Object decode(
 
         sendResponse(channel, remoteAddress, type, buf.getByte(buf.writerIndex() - 2));
 
-        if (type == MSG_POSITION_DATA || type == MSG_ROLLCALL_RESPONSE
+        if (type == MSG_RFID_DATA) {
+            Position position = new Position(getProtocolName());
+
+            DeviceSession deviceSession = getDeviceSession(channel, remoteAddress, "1" + id, id);
+            if (deviceSession == null) {
+                return null;
+            }
+            position.setDeviceId(deviceSession.getDeviceId());
+
+            int subSignal = buf.readUnsignedByte();
```

Comment:
What is sub signal?

Reply:
Wait what? If it's a type, it should definitely not be called `isRFIDData`. And that name is not even consistent with our standards, as we discussed before already.

Reply:
Apologies. I'll use **subType**.